packages/config/config/devices/0x0086/templates/aeotec_template.json

```
@@ -1127,5 +1323,529 @@
+		"unsigned": true,
+		"readOnly": true,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Load",
+				"value": 1
+			},
+			{
+				"label": "Association group 3",
+				"value": 2
+			},
+			{
+				"label": "Load and association group 3",
+				"value": 3
+			}
+		]
+	},
+	"switch_disconnet_s2": {
+		"label": "Control Destination (S2)",
+		"valueSize": 1,
+		"minValue": 1,
+		"maxValue": 3,
+		"defaultValue": 3,
+		"unsigned": true,
+		"readOnly": true,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Load",
+				"value": 1
+			},
+			{
+				"label": "Association group 4",
+				"value": 2
+			},
+			{
+				"label": "Load and association group 3",
+				"value": 3
+			}
+		]
+	},
+	"dimming_rate_255": {
+		"$import": "../../templates/master_template.json#base_1-255_nounit",
+		"label": "Dimming Rate",
+		"unit": "seconds",
+		"defaultValue": 3
+	},
+	"current_working_mode": {
+		"label": "Current Working Mode",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": true,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Unknown",
+				"value": 0
+			},
+			{
+				"label": "2-wire mode (no neutral)",
+				"value": 1
+			},
+			{
+				"label": "3-wire mode",
+				"value": 2
+			}
+		]
+	},
+	"dimming_principle": {
+		"$import": "../../templates/master_template.json#base_0-1_nounit",
+		"label": "Dimming Principle",
+		"defaultValue": 1,
+		"options": [
+			{
+				"label": "Trailing edge mode",
+				"value": 0
+			},
+			{
+				"label": "Leading edge mode",
+				"value": 1
+			}
+		]
+	},
+	"load_type": {
+		"label": "Load Type",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 3,
+		"defaultValue": 0,
+		"unsigned": true,
+		"readOnly": true,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Unknown",
+				"value": 0
+			},
+			{
+				"label": "Resistive load",
+				"value": 1
+			},
+			{
+				"label": "Capacitive load",
+				"value": 2
+			},
+			{
+				"label": "Inductive load",
+				"value": 3
+			}
+		]
+	},
+	"min_brightness": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Minimum Brightness Level",
+		"defaultValue": 0
+	},
+	"max_brightness": {
+		"$import": "../../templates/master_template.json#base_0-99_nounit",
+		"label": "Maximum Brightness Level",
+		"defaultValue": 99
+	},
+	"load_recognition_method": {
+		"label": "Load Recognition Method",
+		"valueSize": 1,
+		"minValue": 0,
+		"maxValue": 2,
+		"defaultValue": 2,
+		"unsigned": true,
+		"readOnly": false,
+		"writeOnly": false,
+		"allowManualEntry": false,
+		"options": [
+			{
+				"label": "Never recognize the load when power on",
+				"value": 0
+			},
+			{
+				"label": "Only recognize once when first power on",
+				"value": 1
+			},
+			{
+				"label": "Recognize the load once power on",
+				"value": 2
+			}
+		]
```

```suggestion
		"options": [
			{
				"label": "Never",
				"value": 0
			},
			{
				"label": "Only on first power on",
				"value": 1
			},
			{
				"label": "Every power on",
				"value": 2
			}
		]
```
would be shorter or is that not specific enough?